docs/quick-answers/linode-platform/add-caa-dns-records/index.md

```diff
@@ -6,52 +6,51 @@ description: 'This quick answer shows how to create CAA records for domains and
 og_description: 'This quick answer will show you how to create a CAA record for domains and subdomains'
 keywords: ["caa", "dns", "records", "subdomain", "domain"]
 license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
-modified: 2017-11-03
```

Comment:
Removed the modified date frontmatter because we no longer use it in our Hugo template--instead, the modified date is generated from the last git commit for the file